control/timeouts.txt

```diff
@@ -79,10 +78,10 @@ ai_route_unstuck 2
 ai_items_take_delay 1.4
```

Comment:
Could probably also lower this value, I have mine set at 0.3 but I'm not trying to hide my bot